tests/apps/loqus/conftest.py

```diff
@@ -176,3 +189,19 @@ def fixture_loqusdb_duplicate_output() -> bytes:
         b'"GT", "CC", "AC", "AG", "AG", "AA", "GA", "AA", "CC", "GG", "AA", "TT", "AA", '
         b'"GG", "GG", "CC", "AA", "TT", "TT"]}'
     )
+
+
+@pytest.fixture(name="loqusdb_delete_stderr")
+def fixture_loqusdb_delete_stderr() -> bytes:
+    """
+    loqusdb stderr for a successful delete.
+    """
+    return LOQUSDB_DELETE_STDERR
```

Comment:
```suggestion
    return b"""2022-09-22 12:30:07 evacmbp loqusdb.commands.cli[20689] INFO Running loqusdb version 2.6.9
2022-09-22 12:30:07 evacmbp mongo_adapter.client[20689] INFO Connecting to uri:mongodb://None:None@localhost:27017
2022-09-22 12:30:07 evacmbp mongo_adapter.client[20689] INFO Connection established
2022-09-22 12:30:07 evacmbp mongo_adapter.adapter[20689] INFO Use database loqusdb
2022-09-22 12:30:07 evacmbp loqusdb.plugins.mongo.case[20689] INFO Removing case yellowhog from database
2022-09-22 12:30:07 evacmbp loqusdb.utils.delete[20689] INFO deleting variants
2022-09-22 12:30:07 evacmbp loqusdb.utils.delete[20689] INFO Start deleting chromosome 1"""
```
I would skip the extra constant initialization step